src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -33,3 +35,13 @@ md-toolbar {
     }
   }
 }
+
+.kd-logs-toolbar-text {
+  font-family: $font-family;
```

Comment:
Font family is already defined by angular material, so it is safe to remove it from here.